src/prefs/SpectrogramSettings.cpp

```diff
@@ -160,14 +160,18 @@ const EnumValueSymbols &SpectrogramSettings::GetColorSchemeNames()
    static const EnumValueSymbols result{
       // Keep in correspondence with enum SpectrogramSettings::ColorScheme:
       /* i18n-hint: New color scheme for spectrograms */
-      XO("Color (New)") ,
+      { wxT("SpecColorNew"), XO("Color (New)") },
       /* i18n-hint: color scheme from theme for spectrograms */
-      XO("Color (from Theme)") ,
+      { wxT("SpecColorTheme"), XO("Color (from Theme)") },
       /* i18n-hint: grayscale color scheme for spectrograms */
-      XO("Grayscale") ,
+      { wxT("SpecGrayscale"), XO("Grayscale") },
       /* i18n-hint: inverse grayscale color scheme for spectrograms */
-      XO("Inv. Grayscale") ,
+      { wxT("SpecInvGrayscale"), XO("Inv. Grayscale") },
    };
+
```

Comment:
Thank you for these changes.

But now, we are in the period of "string freeze."  That means the message catalog locale/audacity.pot, in which all the XO (and XC) strings are extracted, is in a final form, allowing translators to finish their work.  One of our designers reviewed this project, and specified changes to the user-visible strings.

So you must:

1) change the XO strings above to the XC strings below.  (But leave the wxT strings as they are.)
2) Find these strings in lib-strings/locale/UnusedStrings.h; be sure you have copied them exactly; and delete them from that file.
3) Rebase this branch onto more recent master.  (And while doing that, do an interactive rebase, and make sure all commits in the sequence compile.  I believe the first one does not.  And maybe do some fixups to shorten the list of commits.)
4) Test (using one of the completed languages, such as Dutch) that translation works.  (You change language using the Interface page of Preferences.)
5) Force-push your rebased branch.

'
XC("Color (default)",   "spectrum prefs"),
XC("Color (classic)",   "spectrum prefs"),
XC("Grayscale",         "spectrum prefs"),
XC("Inverse grayscale", "spectrum prefs"),
'

Reply:
Done